metadata/identifier/subscribe_metadata_identifier.go

```diff
@@ -32,3 +47,18 @@ func (mdi *SubscriberMetadataIdentifier) GetIdentifierKey() string {
 func (mdi *SubscriberMetadataIdentifier) GetFilePathKey() string {
 	return mdi.BaseMetadataIdentifier.getFilePathKey(mdi.Revision)
 }
+
+func NewSubscriberMetadataIdentifier(application string, revision string) *SubscriberMetadataIdentifier {
+	return &SubscriberMetadataIdentifier{
+		Revision: revision,
+		MetadataIdentifier: MetadataIdentifier{
+			Application: application,
+			BaseMetadataIdentifier: BaseMetadataIdentifier{
+				ServiceInterface: "",
```

Comment:
hey, guy, pls remember that, if its default value is "", u need not to assign the same value to it again.